bindings/python/htcondor2/_schedd.py

```diff
@@ -42,6 +42,7 @@
     _schedd_retrieve_job_constraint,
     _schedd_retrieve_job_ids,
     _schedd_spool,
+    _schedd_refresh_gsi_proxy,
```

Comment:
Maybe we are constrained by convention, but are these technically "ssl" proxies, not "GSI" proxies?  Should our interfaces reflect that?

Reply:
Probably, but we're constrained by matching the version 1 API.